### PR TITLE
[Static pages] Fix for analytics on COVID FAQ page

### PR DESCRIPTION
**Pull request by ncksllvn**
## Description
This PR is a fix for some analytics being broken on `/coronavirus-veteran-frequently-asked-questions/`. Previously, the table of contents on the page was typed into a WYSIWYG, and we'd create event listeners based on that DOM to capture analytics. However, that WYSIWYG content was removed in favor of a generate table of contents. This caused a runtime error in our analytics initializer. This PR fixes that by switching to the new DOM shape.

https://github.com/department-of-veterans-affairs/va.gov-team/issues/13707

## Testing done
Confirmed via Chrome plugin that data layer is getting updated

## Screenshots
image

## Acceptance criteria
- [ ] Analytics are captured again

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/static-pages/analytics/addJumpLinkListeners.js [hunk 1]
```diff
@@ -3,11 +3,12 @@ import { isAnchor } from './utilities';
 
 export default function addJumplinkListeners() {
   const selectors = {
-    heading: '[data-template="paragraphs/wysiwyg"] #on-this-page',
+    template: '#table-of-contents',
```
Comment (kelsonic) [inline src/applications/static-pages/analytics/addJumpLinkListeners.js:6]: How come you made this change? Seems like a big change! 🙂

File: src/applications/static-pages/analytics/addJumpLinkListeners.js [hunk 2]
```diff
@@ -3,11 +3,12 @@ import { isAnchor } from './utilities';
...
   };
 
-  const jumplinkHeading = document.querySelector(selectors.heading);
-  const container = jumplinkHeading.parentElement;
+  const container = document.getElementById(selectors.template);
+
+  if (!container) return;
```
Comment (kelsonic) [inline src/applications/static-pages/analytics/addJumpLinkListeners.js:11]: ❤️

**Review by kelsonic — APPROVED** [review]
LGTM! 🎉

File: src/applications/static-pages/analytics/addJumpLinkListeners.js (hunk 1 above)
Comment (ncksllvn) [reply src/applications/static-pages/analytics/addJumpLinkListeners.js:6]: I wrote about in the PR description...the table of contents was typed into a WYSIWYG before but now it was changed to use the generated version of table of contents...

image

Comment (kelsonic) [reply src/applications/static-pages/analytics/addJumpLinkListeners.js:6]: Oh wow... was that ever communicated? 😬

Comment (ncksllvn) [reply src/applications/static-pages/analytics/addJumpLinkListeners.js:6]: It was just a change by a content writer. I wouldn't have expected them to communicate the change but I also don't know how I would've wrote this analytics script to capture both DOM variations

**Review by U-DON — APPROVED** [review]

File: src/applications/static-pages/analytics/addJumpLinkListeners.js (hunk 1 above)
Comment (kelsonic) [reply src/applications/static-pages/analytics/addJumpLinkListeners.js:6]: Ahh gotcha 👍 makes sense!

**Pull request merged**